src/tutorials/annotation.md

```diff
@@ -128,7 +161,8 @@ annotations in Excel or another tool.
 Instructions for doing this are available in the
 [Spreadsheet templates tutorial](https://hed-examples.readthedocs.io/en/latest/BidsAnnotationQuickstart.html#spreadsheet-templates).
```

Comment:
A better link is:

https://www.hed-resources.org/en/latest/BidsAnnotationQuickstart.html#spreadsheet-templates

Reply:
Updated in latest [commit](https://github.com/bids-standard/bids-starter-kit/pull/335/commits/ec93015455d74f60ae0ce5cc70341d16b1c7e609).